app/views/admin/r_and_r_items/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>New R &amp; R Item</h1>
+<%= render 'form', admin_r_and_r_item: @admin_r_and_r_item,  :url => admin_create_r_and_r_item_path, :method => :post %>
```

Comment:
Why do you need to pass in `:url => admin_create_r_and_r_item_path, :method => :post `?

method post should be the default anyway. `simple_form_for` should come up with the right url anyway. 

It's possible these args aren't being used at all and you can just remove them. If removing them does break things -- can we use conventional routes to not break things still without these args?

Reply:
Done! They were indeed not needed.